src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -123,7 +124,6 @@ public Tenant save(UUID consortiumId, UUID adminUserId, Tenant tenantDto) {
     return savedTenant;
   }
 
-
   @Override
   public Tenant update(UUID consortiumId, String tenantId, Tenant tenantDto) {
```

Comment:
We can update 'name' and 'code', so could you please verify that we get required exception and all changes rolled back when we try to update to an existing 'code' or/and 'name'?

Reply:
![image](https://github.com/folio-org/mod-consortia/assets/113523904/0e10ddaf-a713-41cd-b47a-f389169d6130)
after db no changes in tenant name
![image](https://github.com/folio-org/mod-consortia/assets/113523904/7c108513-821c-480f-b148-28597f032809)

Reply:
method is not Transactional. in case of error happened in db layer check. it will NOT simply save object to db.

Reply:
in order to prevent future issue relate transaction, made transaction method